api.js

```diff
@@ -1849,4 +1849,37 @@ API.prototype.setErrorGroupCallback = function setErrorGroupCallback(callback) {
   this.agent.errors.errorGroupCallback = callback
 }
 
+/**
+ * Registers a callback which will be used for calculating token counts on Llm events when they are not
+ * available. This function will typically only be used if `ai_monitoring.record_content.enabled` is false
+ * and you want to still capture token counts for Llm events.
+ *
+ * Provided callbacks must return an integer value for the token count for a given piece of content.
+ *
+ * @param {Function} callback - synchronous function called to calculate token count for content.
+ * @example
+ * // @param {string} model - name of model (i.e. gpt-3.5-turbo)
+ * // @param {string} content - prompt or completion response
+ * function tokenCallback(model, content) {
```

Comment:
The value of `content` is extremely varied. I can see us fielding plenty of issues asking what it will be.

Reply:
It must be a string. In the openai cases they are always strings.  I know for langchain they aren't always strings but we don't assign tokens. Looking at bedrock they are strings as well

Reply:
https://github.com/jsumners-nr/node-newrelic-aws-sdk/blob/2b36ae2196983997177e1db7843120ac144d7fa7/tests/versioned/v3/bedrock-chat-completions.tap.js#L614-L627

https://github.com/jsumners-nr/node-newrelic-aws-sdk/blob/2b36ae2196983997177e1db7843120ac144d7fa7/lib/llm/bedrock-response.js#L170-L172

Reply:
it shouldn't be the parsed body. it should be the key in the body that contains the content.